mtp_cashbook/assets-src/stylesheets/components/_subheader.scss

```diff
@@ -1,29 +0,0 @@
-.main-content {
-  padding-top: 0;
-}
-
-.global-subheader {
-  padding-top: 12px;
-  padding-bottom: 12px;
-  background: $grey-3-4;
```

Comment:
Can we use a colour from the existing [govuk colour palette](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss)?